lib/main.dart

```diff
@@ -0,0 +1,148 @@
+import 'dart:async';
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:pocketape/arkit_tracer.dart';
+import 'package:vector_math/vector_math_64.dart';
+
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: MeasurementScreen(),
+    );
+  }
+}
+
+class MeasurementScreen extends StatefulWidget {
+  @override
+  _MeasurementScreenState createState() => _MeasurementScreenState();
+}
+
+class _MeasurementScreenState extends State<MeasurementScreen> {
+  Vector3? _firstValue;
+  Vector3? _lastValue;
+  late StreamSubscription<Vector3>? _subscription;
+  bool get isMeasuring => _subscription != null;
+  double _distance = 0.0;
+  String _replacementResult = '';
```

Comment:
Same as for distance, do not store it inside a variable. Instead make `_adjustMeasurement` return the result and directly call it inside the build method.

Currently there is no value of caching it but it introduces the risk of us missing to update it.